web/src/components/molecules/Workspace/WorkspaceHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Content from "@reearth-cms/components/atoms/Content";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import { useT } from "@reearth-cms/i18n";
+
+export interface Props {
+  handleProjectSearch: (value: string) => void;
+  handleProjectModalOpen: () => void;
+  handleWorkspaceModalOpen?: () => void;
+}
```

Comment:
```suggestion
export type Props = {
  handleProjectSearch: (value: string) => void;
  handleProjectModalOpen: () => void;
  handleWorkspaceModalOpen?: () => void;
}
```
Just a suggestion (you can ignore it) since you are using interface as a type.

Reply:
Please make these changes since we across repos prefer type over interface for props.